lib/scr/screens/ribbon/ribbon_screen.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class RibbonScreen extends StatefulWidget {
+  @override
+  _RibbonScreenState createState() => _RibbonScreenState();
+}
+
+class _RibbonScreenState extends State<RibbonScreen> {
+  List<String> myimages = [
```

Comment:
Когда вы даёте название переменной, используйте `snakeCase` для повышения читаемости кода:
```suggestion
  List<String> myImages = [
```